tools/grades-updater/src/sanitize-data.ts

```diff
@@ -3,10 +3,11 @@ import { EOL } from "os";
 import { basename, resolve } from "path";
```

Comment:
Maybe we could flip an ESLint rule for this, but I personally like to specify `node` whenever possible.

```suggestion
import { basename, resolve } from "node:path";
```